main.go

```diff
@@ -5,7 +5,6 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"io/ioutil"
 	"log"
```

Comment:
The import for 'io/ioutil' is now unused; this could be cleaned up to maintain a tidy codebase.